experiment_buddy/utils.py

```diff
@@ -9,6 +9,8 @@
 import git
 from funcy import log_durations
 
+is_running_remotely = "SLURM_JOB_ID" in os.environ.keys() or "BUDDY_IS_DEPLOYED" in os.environ.keys()
```

Comment:
make it a function?
otherwise i don't see the point of moving it away

Reply:
I just needed the value in two different places. Function created